seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style\\s*?=\\s*[\\\"]{1}\\s*\\s*\\S*;*\\s*color\\s*?:\\s*?white\\s*;*\\s*[\\\"]{1}/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
If I use single quotes like `'color: white'`, the test will fail.

We do not need the curly-braced quantifies for this.

 `\s{1}` is the exact same as `\s`.

The curly-braced quantifiers are handy if you need to match an exact number of something, no more, no less. `\s{10}` is more readable than `\s\s\s\s\s\s\s\s\s\s`